templates/private_endpoint.json

```
@@ -0,0 +1,202 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string"
+        },
+        "privateEndpointName": {
+            "type": "string"
+        },
+        "privateLinkResourceId": {
+            "type": "string"
+        },
+        "targetSubResource": {
+            "type": "array",
+            "defaultValue": ["blob"]
+        },
+        "subnet": {
+            "type": "string"
+        },
+        "vnet": {
+            "type": "string"
+        },
+        "resourceGroup": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]"
+        },
+        "subscriptionId": {
+            "type": "string"
+        },
+        "subnetDeploymentName": {
+            "type": "string",
+            "defaultValue": "[concat(resourceGroup().name, '-sub')]"
```

Minor: This could be confused for "subscription."
```suggestion
            "defaultValue": "[concat(resourceGroup().name, '-subnet')]"
```